newshour-namedentity-wikipedialink/readme.md

```diff
@@ -1,19 +1,124 @@
+While this can be useful, that usefulness is impacted by the fact that the same word or phrase can mean different things. 
+For example, "President Bush" and "the Bush administration" can refer to [George H.W. Bush senior](https://en.wikipedia.org/wiki/George_H._W._Bush) or his son [George W. Bush](https://en.wikipedia.org/wiki/George_W._Bush). 
+Since there's no way to give different links to annotations that have the same label and spelling, **one set will invariably be wrong**. As far as the annotator knows, there is no workaround.
+
+* **Lack of Authoritative Mention/Article** - Not everything mentioned in broadcasts has a wikipedia article. 
+In those cases, search the web for other acceptable/wikipedia links, but when no such source exists, the annotator must simply move on.  
+## Data Format and `process.py`
+
+### `raw` data
+`.tab` file where each line is a new instance of a named entity. This seems to be a tab spaced text file.
+* Format:
+```
+anno_id_tag#\tdate_time_of_annotation\tsource_transcript.ann_file\tentity_text\tcategory\tduplicate_count\tWikipedia_link(normal_url)
+```
+* Fields: 
+  * `anno_id_tag#` - entry number
+  * `date_time_of_annotation` - _TODO: Unverified what this refers to_
+  * `source_transcript.ann_file` - which GUID's transcript this entry comes from
+  * `entity_text` - text of the named entity
+  * `category` - which category it belongs to
+  * `duplicate_count` - NEW, added by preprocessing before raw, how many times this entry appears probably within one transcript
+  * `Wikipedia_link(normal_url)` - NEW, this is to be annotated now, the relevant human-readable/searchable link to this specific entity. 
+* Example:
+```
+1	2022-11-10 14:38:22	cpb-aacip-507-154dn40c26-transcript.ann	ROBERT MacNEIL	person	1	https://en.wikipedia.org/wiki/Robert_MacNeil	
+2	2022-11-10 14:38:54	cpb-aacip-507-154dn40c26-transcript.ann	U.S.	location	4	https://en.wikipedia.org/wiki/United_States	
+3	2022-11-10 14:39:11	cpb-aacip-507-154dn40c26-transcript.ann	Health and Human Services Secretary Margaret Heckler	person	1	https://en.wikipedia.org/wiki/Margaret_Heckler	
+```
+> [!Note]  
+> Different spellings are not considered duplicates. eg. in the [raw file](https://github.com/clamsproject/aapb-annotations/blob/main/newshour-namedentity-wikipedialink/221201-aapb-collaboration-21/annotations.tab)
+line 15 is "Judy Woodruff" while line 54 is "JUDY WOODRUFF".  
+> Also, see below for single link only issue.  
+
+### [`process.py`](process.py)
+_TODO: Needs summary._  
+Please see the docstring of [`process.py`](process.py).  
+_TODO: It is suggested to possibly place code that is required for pre-processing for raw to either use "preprocessing" in the name,
+or be placed into a directory to avoid tangling with the expected process code that transforms raw -> post-process -> gold._  
```

Comment:
As suggested, it seems though that gold-retriever is used and this might be correctly only raw->gold processing files while the pre-processing files are in a different repository linked above.

Reply:
Removing suggestion. Its unclear if the goldretriever is also part of the process.py (seems like it is).